drivers/hmis/app/graphql/types/hmis_schema/inventory.rb

```diff
@@ -26,7 +26,7 @@ def self.configuration
     hud_field :youth_bed_inventory
     hud_field :ch_bed_inventory
     hud_field :other_bed_inventory
-    hud_field :es_bed_type, HmisSchema::Enums::BedType
+    hud_field :es_bed_type, HmisSchema::Enums::Hud::BedType
```

Comment:
Should the enum be EsBedType?  (I'm guessing there aren't other bed types, so probably not.)

Reply:
It's called BedType in HUD. I think its easier to keep the names the same as HUD. There was one I had to change, `Event`->`EventType`, since we already have an `Event` type and Graphql didn't like having an Enum with the same name.
![Screen Shot 2022-11-16 at 9 08 15 AM](https://user-images.githubusercontent.com/5333982/202202202-2d7c4f4b-462f-445c-9e9f-67761ce2b074.png)